ng-dev/pr/common/validation/assert-enforce-tested.ts

```diff
@@ -44,18 +53,34 @@ function pullRequestRequiresTGP(pullRequest: PullRequestFromGithub): boolean {
   return pullRequest.labels.nodes.some(({name}) => name === requiresLabels.REQUIRES_TGP.name);
 }
 
+export class PullRequestComments {
```

Comment:
Can this just be a function instead of class that gets instantiated and then executed?

Reply:
The reason for the class is so testing has a static create function to mock out the call to the API.